auditbeat/docs/reload-configuration.asciidoc

```diff
@@ -1,6 +1,10 @@
 [id="{beatname_lc}-configuration-reloading"]
 == Reload the configuration dynamically
 
+++++
+<titleabbrev>Config Reloading</titleabbrev>
+++++
+
 beta[]
 
 You can configure {beatname_uc} to dynamically reload configuration files when
```

Comment:
See other comment. This file is not included anywhere in the auditbeat docs and therefor not built.

Reply:
good catch. Added!